tutorials/developer-tutorials/createNewIdentityAttributes.md

```diff
@@ -0,0 +1,168 @@
+ 
+> Command Palette alternative: `Package Open Project`
+
+3. You should see a new package on the list, `carContract@0.0.1`, if everything went well.
+
+</details>
+
+<details>
+<summary><b>Install the smart contract</b></summary>
+
+To do this...
+
+1. In the `Fabric Environments` panel, look for `+ Install` (it's under Smart Contracts > Installed) and click it.
+
+2. You'll be asked to choose a package to install. Pick `carContract@0.0.1`.
+
+You should see `carContract@0.0.1` appear under the Smart Contracts > Installed list.
+
+> Command Palette alternative: `Install Smart Contract`
+
+That's it - job done! Next up, we'll instantiate the smart contract...
+
+</details>
+
+
+<details>
+<summary><b>Instantiate the smart contract</b></summary>
+
+To instantiate our installed contract...
+
+1. In the `Fabric Environments` panel, look for `+ Instantiate` (it's under Smart Contracts > Instantiated) and click it.
+
+2. You'll be asked to choose a smart contract to instantiate. Pick `carContract@0.0.1`.
+
+3. You'll be asked what function to call. If you wanted to use a specific function as part of your instantiate, you could enter something here.  We'll see that happen in future tutorials, but for now just hit `Enter` to skip this step.
+
+4. You'll be asked if you want to provide a private data configuration file. For this tutorial just click `No`, in future tutorials we will explain more about this.
+
+Instantiation can take a while longer than install - watch out for the success message and `carContract@0.0.1` appearing in the Smart Contracts > Instantiated list to confirm it's worked!
+
+> Command Palette alternative: `Instantiate Smart Contract`
+</details>
+
+4. To test that attribute-based access control works, we can connect to the gateway by pressing `Local Fabric` under the `FABRIC GATEWAYS` panel. You will now be asked which identity you wish to connect with - choose `Jack` (created in our previous tutorial). 
+> Command Palette alternative: `Connect Via Gateway`
+
+5. To execute a function, navigate to `mychannel`, `carContract@0.0.1` under the `FABRIC GATEWAYS` panel. Here, you will see a list of all the transactions you can carry out. For the scope of this tutorial, we will be submitting the `createCar` transaction (right-click on the transaction and `Submit Transaction`). Say we want to create a car with the arguments `["001", "Model X"]`, it will show the error message `Error: You must be a manufacturer to carry out this transaction!` because Jack is not a manufacturer!
+
+6. However, you will be able to create your desired car successfully if you connect to the gateway with the identity that has the required attribute. To do this, disconnect from the gateway (by hovering on the `FABRIC GATEWAYS` panel and pressing the button that looks like an exit symbol). Now, repeat step 5 and connect with `Arium` as the chosen identity and the car (Model X) will be created.
```

Comment:
I think you should just give them the exact instructions here

Reply:
Which step are you referring to?

Reply:
Fixed.